Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj

```diff
@@ -117,20 +117,22 @@
 				<Property Name="Bld_excludeInlineSubVIs" Type="Bool">true</Property>
 				<Property Name="Bld_excludeLibraryItems" Type="Bool">true</Property>
 				<Property Name="Bld_excludePolymorphicVIs" Type="Bool">true</Property>
-				<Property Name="Bld_localDestDir" Type="Path">../builds/NI_AB_PROJECTNAME/Keysight 34401A DMM Measurement</Property>
-				<Property Name="Bld_localDestDirType" Type="Str">relativeToCommon</Property>
+				<Property Name="Bld_localDestDir" Type="Path">/c/build_output/keysight</Property>
 				<Property Name="Bld_modifyLibraryFile" Type="Bool">true</Property>
 				<Property Name="Bld_postActionVIID" Type="Ref">/My Computer/Keysight 34401A DMM Measurement.lvlib/Advanced/Build Assets/Post-Build Action.vi</Property>
 				<Property Name="Bld_previewCacheID" Type="Str">{10C7A19E-ADBC-4742-B617-E2F1EE61FB5A}</Property>
+				<Property Name="Bld_version.build" Type="Int">3</Property>
 				<Property Name="Bld_version.major" Type="Int">1</Property>
 				<Property Name="Destination[0].destName" Type="Str">Keysight 34401A DMM Measurement.exe</Property>
-				<Property Name="Destination[0].path" Type="Path">../builds/NI_AB_PROJECTNAME/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.exe</Property>
+				<Property Name="Destination[0].path" Type="Path">/c/build_output/keysight/NI_AB_PROJECTNAME.exe</Property>
+				<Property Name="Destination[0].path.type" Type="Str">&lt;none&gt;</Property>
 				<Property Name="Destination[0].preserveHierarchy" Type="Bool">true</Property>
 				<Property Name="Destination[0].type" Type="Str">App</Property>
 				<Property Name="Destination[1].destName" Type="Str">Support Directory</Property>
-				<Property Name="Destination[1].path" Type="Path">../builds/NI_AB_PROJECTNAME/Keysight 34401A DMM Measurement/data</Property>
+				<Property Name="Destination[1].path" Type="Path">/c/build_output/keysight/data</Property>
```

Comment:
Ditto with regards to `NI_AB_PROJECTNAME`.

Reply:
It looks like this is inconsistently applied across the projects. Please update all of the projects so they are consistent in this regard. It looks like `NI_AB_PROJECTNAME` is a symbolic path that LV will recognize so I imagine you would enter the path using that symbolic constant directly in the properties dialog for the build spec. Or you can probably just directly edit the XML if that is easier.

https://labviewwiki.org/wiki/Application_Builder